src/database/migrations/20201129004549-create-user.js

```diff
@@ -51,6 +51,10 @@ export async function up(queryInterface, Sequelize) {
 			type: Sequelize.STRING,
 			defaultValue: 'Requester',
 		},
+		lineManager: {
```

Comment:
we don't update database like that you must create another migration to update the existing one!

Reply:
Resolved